### PR TITLE
Add option : g:vimwiki_table_reduce_last_col. Used to not expand tabl…

### PR DESCRIPTION
**Pull request by tinmarino**
…e last cells in order to avoid large, useless wraps

Update from #559

### CODE REVIEW

**Comment by ranebrown** [comment]
This doesn't seem to be working correctly for me. @tinmarino can you confirm the behavior is correct when setting `let g:vimwiki_table_reduce_last_col = 1`?

**Comment by ranebrown** [comment]
Disregard my previous comment. I had my runtime path messed up from testing something else. Here is another comparison image in case anyone is interested. The top table uses the new option to get rid of extra whitespace.

image

File: doc/vimwiki.txt [hunk 1]
```diff
@@ -2719,6 +2719,20 @@ Value           Description~
 Default: 1
 
 
+------------------------------------------------------------------------------
+*g:vimwiki_table_reduce_last_col*
+
+If set, the last column spearator will not be expanded to fill the cell.
```
Comment (bewuethr) [inline doc/vimwiki.txt]: Typo, "spearator" instead of "separator"

Comment (ranebrown) [reply doc/vimwiki.txt]: Fixed. Thanks!

**Comment by tinmarino** [comment]
Thank you so much  @ranebrown for this clean integration.
My first open source commit (on an usefull project).

**Pull request merged**